packages/valory/skills/hello_world_abci/models.py

```diff
@@ -62,9 +60,8 @@ class HelloWorldParams(BaseParams):
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         """Initialize the parameters."""
+        self.hello_world_string: str = self._ensure("hello_world_message", kwargs, str)
         super().__init__(*args, **kwargs)
-        self.hello_world_string = self._ensure("hello_world_message", kwargs)
```

Comment:
not possible - at this point it's frozen - swapping solves it

Reply:
Maybe it makes sense to include this in the raised exception message?

Reply:
Hm feels inappropriate

Reply:
It would be good to hint somehow how it could possibly be resolved.